### PR TITLE
Move test-as-you-commit text to Success Criteria

### PR DESCRIPTION
**Pull request by xfq**
Partial fix for #88.

### CODE REVIEW

File: DASCharter-2020.html [hunk 1]
```diff
@@ -922,6 +922,46 @@ <h2 id="success-criteria">
           meaningfully consent to the use of sensitive data, it should not
           advance those specifications.
         </p>
+
+        <div>
+          <h3>
```
Comment (anssiko) [inline DASCharter-2020.html]: I suggest drop this header.

**Review by anssiko — CHANGES_REQUESTED** [review]

File: DASCharter-2020.html (hunk 1 above)
Comment (xfq) [reply DASCharter-2020.html]: Removed.

File: DASCharter-2020.html [hunk 2]
```diff
@@ -960,8 +956,6 @@ <h3>
               "https://github.com/web-platform-tests/wpt/issues?utf8=%E2%9C%93&amp;q=label%3Atype%3Auntestable%20">type:untestable</a>'
               label to follow up later.
             </p>
-          </blockquote>
```
Comment (anssiko) [inline DASCharter-2020.html]: I think we need to keep the closing tag.

**Review by anssiko — APPROVED** [review]
LGTM with a nit.

File: DASCharter-2020.html [hunk 3]
```diff
@@ -1026,45 +1060,6 @@ <h3>
...
-              change.
-            </p>
-            <p>
-              If testing is not practical due to web-platforms-tests
-              limitations, please explain why and if appropriate file an issue
-              with the '<a href=
-              "https://github.com/web-platform-tests/wpt/issues?utf8=%E2%9C%93&amp;q=label%3Atype%3Auntestable%20">type:untestable</a>'
-              label to follow up later.
-            </p>
-          </blockquote>
```
Comment (anssiko) [inline DASCharter-2020.html:1066]: Keep.

**Review by anssiko — APPROVED** [review]

File: DASCharter-2020.html (hunk 2 above)
Comment (xfq) [reply DASCharter-2020.html]: Fixed. Thank you for catching this!

**Pull request merged**